.travis.yml

```diff
@@ -9,6 +9,7 @@ node_js:
   - "0.10"
 env:
   - TEST_DIR=packages/fluxible
+  - TEST_DIR=packages/fluxible-reducer-store
```

Comment:
Looks like it's not working as I expected and is still testing everything for every TEST_DIR. Will look into whether I can get this working correctly.